### PR TITLE
Add support for actions variables

### LINKED ISSUE #2624: Add support for configuration variables in workflows
Recently GitHub added support for configuration variables in github actions workflows: https://github.blog/changelog/2023-01-10-github-actions-support-for-configuration-variables-in-workflows/

Please add support for creating configuration variables. Configuration variables can be created at the organization, repository, or environment level so support for all 3 of these scopes should be added.

### PR DESCRIPTION
**Pull request by FedeDP**
https://docs.github.com/en/rest/actions/variables

Fixes #2624 

* Added `actions_variables.go` 
* Added `actions_variables_test.go`

### CODE REVIEW

File: github/actions_variables.go
Comment (gmlewis) [inline github/actions_variables.go:26]:
```suggestion
// ActionsVariables represents one item from the ListVariables response.
```

File: github/actions_variables.go
Comment (gmlewis) [inline github/actions_variables.go:13]:
```suggestion
// ActionsVariable represents a repository action variable.
```

File: github/actions_variables_test.go
Comment (gmlewis) [inline github/actions_variables_test.go:179]:
```suggestion
	const methodName = "DeleteRepoVariable"
```

File: github/actions_variables.go
Comment (gmlewis) [inline github/actions_variables.go:23]:
```suggestion
	CreatedAt  *Timestamp `json:"created_at,omitempty"`
	UpdatedAt  *Timestamp `json:"updated_at,omitempty"`
	Visibility *string    `json:"visibility,omitempty"`
	// Used by ListOrgVariables and GetOrgVariables
	SelectedRepositoriesURL *string `json:"selected_repositories_url,omitempty"`
	// Used by UpdateOrgVariable and CreateOrgVariable
	SelectedRepositoryIDs *SelectedRepoIDs `json:"selected_repository_ids,omitempty"`
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @FedeDP!
It looks great and with a few minor tweaks we should be ready for a second LGTM+Approval before merging.

File: github/actions_variables.go
Comment (FedeDP) [reply github/actions_variables.go:26]: I am not sure, given that `type Secrets` has 
```
// Secrets represents one item from the ListSecrets response.
```
comment. Is it ok?

Comment (gmlewis) [reply github/actions_variables.go:26]: Ah, that is unfortunate.  You are saying that "Secrets" should be "ActionsSecrets", correct?
I agree, but let's not make that breaking change here.

File: github/actions_variables.go
Comment (FedeDP) [reply github/actions_variables.go:23]: As said above, i strictly followed `actions_secrets.go` here! Do you want me to change this anyway?

Comment (gmlewis) [reply github/actions_variables.go:23]: Yes, I believe these changes would be good and/or needed here.

It may be that I allowed errors into actions_secrets.go during my code review, but I hope that is not the case.
I looked at the docs for actions variables, and I believe these changes are necessary, so please let's apply these changes to this PR.

Comment (gmlewis) [reply github/actions_variables.go:23]: And please remember to run "go generate ./..." and push the results once you finish making the changes.

File: github/actions_variables.go
Comment (gmlewis) [reply github/actions_variables.go:26]: Both, and everywhere that it is used, please.

**Comment by FedeDP** [comment]
Should've fixed everything ;) Thank you!

**Review by gmlewis — APPROVED** [review]
Thank you, @FedeDP !
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

**Comment by FedeDP** [comment]
I merged master into this branch.

**Comment by gmlewis** [comment]
> LGTM!

Thank you, @valbeat !
Merging.

**Pull request merged**
